packages/devtools_app/lib/src/debugger/debugger_model.dart

```diff
@@ -702,18 +751,63 @@ class Variable extends TreeNode<Variable> {
     return Variable._(null, null, text);
   }
 
+  factory Variable.grouping(
+    GenericInstanceRef ref, {
+    @required int offset,
+    @required int count,
+  }) {
+    return Variable._(
+      null,
+      ref,
+      '[$offset - ${offset + count - 1}]',
+      offset: offset,
+      childCount: count,
+    );
+  }
+
+  static const MAX_CHILDREN_IN_GROUPING = 100;
+
   final String text;
   final String name;
   GenericInstanceRef get ref => _ref;
   GenericInstanceRef _ref;
 
+  int get offset {
```

Comment:
this can be a one liner: `int get offset => _offset ?? 0;`

Reply:
also add dart doc explaining what offset is

Reply:
Nice! Done